saint-app/domain/src/interactors/saint_mutation.rs

```diff
@@ -1,16 +1,49 @@
 use crate::boundaries;
-use crate::boundaries::{SaintDbGateway, SaintMutationRequest, SaintMutationResponse};
+use crate::boundaries::{
+    SaintDbGateway, SaintDbRequest, SaintMutationRequest, SaintMutationResponse,
+};
 use async_trait::async_trait;
 use futures::executor::block_on;
+use uuid::Uuid;
 
 pub struct SaintMutationInteractor {
     db_gateway: Box<dyn SaintDbGateway>,
 }
 
 impl boundaries::SaintMutationInputBoundary for SaintMutationInteractor {
```

Comment:
we should use async while we can, we can async here, so, let's use it instead of blocking every where.